index.js

```diff
@@ -75,24 +86,28 @@ function spawnClangFormat(args, done, stdio) {
                                           args.concat(chunk),
                                           {stdio: stdio});
             clangFormatProcess.on('close', function(exit) {
-              if (exit !== 0) callback(exit);
-              else callback(null, exit);
+              if (exit !== 0) callback(errorFromExitCode(exit));
+              else callback();
             });
           };
         }),
-        function(err, results) {
-          if (err) done(err);
+        function(err) {
+          if (err) {
+            done(err);
+            return;
+          }
           console.log('\n');
           console.log('ran clang-format on',
                       files.length,
                       files.length === 1 ? 'file' : 'files');
-          done(results.shift() || 0);
+          done();
         });
     });
   } else {
     var clangFormatProcess = spawn(nativeBinary, args, {stdio: stdio});
     clangFormatProcess.on('close', function(exit) {
-      if (exit) done(exit);
+      if (exit) done(new Error('clang-format exited with exit code ' + exit + '.'));
```

Comment:
use `errorFromExitCode` here, too?

Reply:
Oops -- this was whole point of having `errorFromExitCode` 🙃